foundation-ui/utils/ReactUtil.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+
+const ReactUil = {
+  /**
+   * Wrap React elements
+   *
+   * @param {Array.<ReactElement>|ReactElement} elements
+   * @param  {function|String} wrapper component
+   * @param {boolean} [alwaysWrap]
+   *
+   * @returns {ReactElement} wrapped react elements
+   */
+  wrapElements(elements, wrapper = 'div', alwaysWrap = false) {
```

Comment:
I think we'll need to be able to set attributes on the element itself, for example, for a <tr>, we could want to add classes to it.